packages/quantic/force-app/main/default/lwc/quanticRecommendationList/e2e/pageObject.ts

```diff
@@ -0,0 +1,51 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaClickEvent} from '../../../../../../playwright/utils/requests';
+
+export class RecommendationListObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get recommendationList(): Locator {
+    return this.page.locator('c-quantic-recommendation-list');
+  }
+
+  get recommendationLinks(): Locator {
+    return this.page.locator('c-quantic-result-link a');
+  }
+
+  get firstRecommendationLink(): Locator {
```

Comment:
That's not scalable to have a selector that just selects the first link.
A more scalable approach would be to have a function `getRecommendationLinkByIndex(index: number)`. instead of limiting you to just the first you can now pick any index that suits your tests.